### ISSUE TITLE
Oberon vs Alto

### ISSUE THREAD

**Issue by ghost** [livingcomputermuseum/ContrAlto#13]
Sorry if this sound noobish

Did ETH Zurich's Oberon project kind of adopted a lot of ideas from Alto? They have a very good Win application emulator. Could ContrAlto be implemented to use Oberon's Win GUI?

**Comment by tommythorn** [comment]
Wirth interned at PARC, so YES.  He writes something to the extent of wanting such a computer for himself.

Tommy

**Comment by tommythorn** [comment]
For the second part, no, there no need or benefit.  There are already good simulators for both and they two architectures have nothing in common.

Tommy

**Comment by tommythorn** [comment]
Correction: I wrote interned but meant visited — the account can be found with a trivial search:

https://amturing.acm.org/award_winners/wirth_1025774.cfm
“In 1976, influenced by his exposure to the new workstation technology he used while on a sabbatical leave at Xerox’s Palo Alto Research Center (PARC), he led a project at ETH to build a new graphical workstation. Named Lilith, it was a complete computing environment, ...”

Tommy

**Comment by ghost** [comment]
Yeah, potatoe, potatoe. The Wikipedia article describes it as a "sabbatical". Go figure.

Thanks. I was checking out BravoX in the ContrAlto, and it dawned on me. The bitsavers PDFs need to be optimized though, they are way too heavy to navigate.
